components/surveys/forms/admin-survey-form.tsx

```
@@ -0,0 +1,361 @@
+					label="Select Item"
+					placeholder="Choose an item for this survey"
+				/>
+			)}
+
+
+			{/* Survey JSON */}
+			<div>
+				<div className="flex items-center justify-between mb-2">
+					<label className="block text-sm font-medium">
+						Survey Definition (JSON) <span className="text-red-500">*</span>
+					</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)